src/tweenable.rs

```diff
@@ -1203,7 +1345,16 @@ mod tests {
             |world: &mut World, mut events: Mut<Events<TweenCompleted>>| {
                 let transform = world.get_mut::<T>(entity).unwrap();
                 let mut target = ComponentTarget::new(transform);
-                tween.tick(duration, &mut target, entity, &mut events)
+                // let command_queue = &mut CommandQueue::default(); // todo
+                // let mut asd = Commands::new(command_queue, world);
+                tween.tick(
+                    duration,
+                    &mut target,
+                    entity,
+                    &mut events,
+                    // passing dummy values to let things compile
+                    &mut Commands::new(&mut CommandQueue::default(), &World::default()),
```

Comment:
You can use (from Bevy's own code):

```rust
let mut command_queue = CommandQueue::default();
let mut commands = Commands::new(&mut command_queue, &world);
```

That way we could even inspect the command queue to confirm the sytem ran (or, apply it to the world and make sure the system ran by setting a flag when it does and checking the flag after the fact).

Reply:
Have you tried this? Borrow checker not letting me do this.